build.sbt

```diff
@@ -250,7 +250,13 @@ lazy val munitScalacheck = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(
     moduleName := "munit-scalacheck",
     sharedSettings,
-    libraryDependencies += "org.scalacheck" %%% "scalacheck" % "1.15.2"
+    libraryDependencies += {
+      val partialVersion = CrossVersion.partialVersion(scalaVersion.value)
+      if (isNotScala211(partialVersion))
+        "org.scalacheck" %%% "scalacheck" % "1.15.3"
+      else
+        "org.scalacheck" %%% "scalacheck" % "1.15.2"
```

Comment:
@tgodzik / @olafurpg I wonder when will it be a good time to start dropping 2.11 🤔

Reply:
If ScalaCheck has dropped 2.11 then I won’t oppose dropping it.

Reply:
We use 2.11 in Metals so maybe it’s worth supporting for that usecase alone.

Reply:
Right. I would then probably defer it to when Metals will drop 2.11 (it seems fair to do it in a bit, since native is now supporting more recent Scala versions)

Reply:
I think 2.11 is still quite heavily used by companies on spark. I have mixed feelings about having deprecated 2.11 in Metals. I haven’t looked at the download numbers to confirm.

Reply:
Yeah, let's look at the numbers and see what's the situation. On one end, those numbers are indicative, but on the other they're not independent: if the ecosystem nudges you to move forward, you probably do, at some point. And Metals is part of the ecosystem.